grails-app/assets/javascripts/measurementSetup/_createJob.js

```diff
@@ -61,22 +58,8 @@ OpenSpeedMonitor.MeasurementSetupWizard.CreateJobCard = (function () {
         validateCronExpression();
     }
 
-    var validateJobNameInput = function () {
-        var currentJobName = jobNameInput.val();
-        var isDuplicateName = isExistingJobName(currentJobName);
-
-        $("#jobNameFormGroup").toggleClass("has-error", !currentJobName);
-        jobNameHelpBlock.toggleClass("hidden", !isDuplicateName);
```

Comment:
The `jobNameHelpBlock` is now unused. You can remove the variable, the markup and both translations.